ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTestFuns.scala

```diff
@@ -399,14 +403,19 @@ trait AbstractHttpServiceIntegrationTestFuns
       val MultiPartyContract: Id = domain.TemplateId(None, "Test", "MultiPartyContract")
     }
     object Account {
-      val Account: Id = domain.TemplateId(None, "Account", "Account")
+      val Account: TId = CtId.Template(None, "Account", "Account")
     }
     object User {
       val User: Id = domain.TemplateId(None, "User", "User")
     }
     object IIou {
-      val IIou: Id = domain.TemplateId(None, "IIou", "IIou")
+      val IIou: IId = CtId.Interface(None, "IIou", "IIou")
     }
+
+    def unsafeCoerce[Like[T] <: CtId[T], T](ctId: CtId[T])(implicit
+        Like: CtId.Like[Like]
```

Comment:
minor: any reason for `Like` instead of `like`?

Reply:
One pattern for typeclass evidence is to name the evidence exactly after the type parameter it describes, [for example](https://github.com/scalaz/scalaz/blob/d0a8b49c3de17d90f3dafae6089aa3d0eb7594ea/core/src/main/scala/scalaz/Traverse.scala#L227). I still instinctively reach for `ev` like any Scala programmer, but this is a habit worth redirecting.